cmd/whereabouts_test.go

```diff
@@ -57,7 +57,7 @@ func AllocateAndReleaseAddressesTest(ipVersion string, ipamConf *whereaboutstype
 	Expect(ipamConf.IPRanges).NotTo(BeEmpty())
 	wbClient := *kubernetes.NewKubernetesClient(
 		fake.NewSimpleClientset(
-			ipPool(ipamConf.IPRanges[0].Range, podNamespace)),
+			ipPool(ipamConf.IPRanges[0].Range, podNamespace, ipamConf.NetworkName)),
```

Comment:
I'm confused; shouldn't this the the "global" pool ?

Reply:
This passes through whatever is configured in the test in the `ipamConf` -- if a test uses `""` it uses the "global" (unnamed?) pool. But this is consistent as currently "normalizing" with an empty Name will give the unnamed pool.

Reply:
OK, I guess this is explicit enough.